packages/stream_chat/lib/src/api/channel.dart

```diff
@@ -413,11 +410,11 @@ class Channel {
   }
 
   /// Deletes the [message] from the channel.
-  Future<EmptyResponse> deleteMessage(Message message) async {
+  Future<EmptyResponse?> deleteMessage(Message message) async {
     // Directly deleting the local messages which are not yet sent to server
     if (message.status == MessageSendingStatus.sending ||
         message.status == MessageSendingStatus.failed) {
-      state.addMessage(message.copyWith(
+      state!.addMessage(message.copyWith(
```

Comment:
can we create a global assertion for the state and check it at the beginning of every function before continuing the execution, just to be sure we access this with a null state.